Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/GameObject.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace sne
 {
-	GameObject::GameObject(): components(),model(1.0f)
+	GameObject::GameObject(): components(), model(1.0f), name("GameObject")
```

Comment:
Is it to print it for gebugging ?

Reply:
It might be usefull in the future if we want to communicate between objects. For now it is only for debugging